### DISCUSSION TITLE
Disable notifications

### DISCUSSION THREAD

**Discussion by oaklsb** [ciderapp/Cider#1790] [Feature Request]
Is there any way to disable notifications completely? I am using Cider as a flatpak app on Ubuntu 22.04 and each time a new song starts playing I get a notification which is pretty annoying. I see no obvious way to disable it.

**Comment by SimonSwopes** [comment]
I had a similar issue on fedora 40. The solution for me was open the app and click on the cider icon in the top left corner. From there: "Settings" -> "Connectivity" -> Then you can toggle "Playback Notifications"
